editor/src/messages/tool/tool_messages/spline_tool.rs

```diff
@@ -264,8 +265,20 @@ fn add_spline(tool_data: &SplineToolData, global_tool_data: &DocumentToolData, s
 	let subpath = bezier_rs::Subpath::new_cubic_spline(points);
 
 	let layer_path = tool_data.path.clone().unwrap();
+	let manipulator_groups = subpath.manipulator_groups().to_vec();
 	graph_modification_utils::new_vector_layer(vec![subpath], layer_path.clone(), responses);
 
+	// Set the manipulator groups to have their handle angles mirrored by default
+	for manipulator_group in manipulator_groups {
+		responses.add(GraphOperationMessage::Vector {
+			layer: layer_path.clone(),
+			modification: VectorDataModification::SetManipulatorHandleMirroring {
+				id: manipulator_group.id,
+				mirror_angle: true,
+			},
+		});
+	}
```

Comment:
@0HyperCube this is now duplicate code in two places (also ellipse_tool.rs:135), is there a good place this could be extracted to to avoid the duplication? @Chase-Percy you're also welcome to try and find somewhere before he responds with a suggestion.

Reply:
Perhaps `editor/src/messages/tool/common_functionality/graph_modification_utils.rs`?

Reply:
Sure, I can refactor it into there if you would like to go ahead with that.

Reply:
I wasn't 100% sure how to name the function without it being massive. Feel free to suggest something else. [243f37b](https://github.com/GraphiteEditor/Graphite/pull/1182/commits/243f37b8fc2497729d1eddaa71371067b60bb6b5)

Reply:
Perhaps `mirror_all_handles`?